tests/test_SklearnNaiveBayesConverter.py

```diff
@@ -25,15 +25,15 @@ def _fit_model_multiclass_classification(self, model):
         model.fit(X, y)
         return model, X.astype(numpy.float32)
 
-    @unittest.skipIf(StrictVersion(onnx.__version__) < StrictVersion('1.2'), 'Need Greater Opset 7')
+    @unittest.skipIf(StrictVersion(onnx.__version__) < StrictVersion('1.3'), 'Need Greater Opset 8')
     def test_model_multinomial_nb_binary_classification(self):
         model, X = self._fit_model_binary_classification(MultinomialNB())
         model_onnx = convert_sklearn(model, 'multinomial naive bayes', [('input', FloatTensorType([1, 4]))])
         self.assertIsNotNone(model_onnx)
         dump_data_and_model(X, model, model_onnx, basename="SklearnBinMultinomialNB-OneOff",
                             allow_failure="StrictVersion(onnxruntime.__version__) <= StrictVersion('0.1.3')")
 
-    @unittest.skipIf(StrictVersion(onnx.__version__) <= StrictVersion('1.3'), 'Need Greater Opset 9')
+    @unittest.skip(reason="Type 'tensor(float)' of input parameter (input) of operator (ConstantOfShape) in node (ConstantOfShape) is invalid.")
```

Comment:
Is it possible to split it over multiple lines, it requires scrolling to see the entire line.

Reply:
Done